ext/dapr-ext-fastapi/dapr/ext/fastapi/app.py

```diff
@@ -83,7 +85,8 @@ def decorator(func):
                 "pubsubname": pubsub,
                 "topic": topic,
                 "route": event_handler_route,
-                "metadata": metadata
+                "metadata": metadata,
+                **({"deadLetterTopic": dead_letter_topic} if dead_letter_topic else {})

```

Comment:
I didn't realize this syntax can be used in this context. TIL -- very useful!

I'd probably have written another.. `if dead_letter_topic append...`

Reply:
Works for me!